release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -430,6 +430,19 @@ The Network Observability Operator releases updates independently from the {prod
 
 With this release, the ports that serve metrics for the Cluster Machine Approver Operator and Cluster Cloud Controller Manager Operator use the Transport Layer Security (TLS) protocol for additional security. (link:https://issues.redhat.com/browse/OCPCLOUD-2272[*OCPCLOUD-2272*], link:https://issues.redhat.com/browse/OCPCLOUD-2271[*OCPCLOUD-2271*])
 
+[id="ocp-4-15-cluster-cloud-controller-manager-operator"]
+=== Cloud controller manager for Google Cloud Platform
+
+The Kubernetes community plans to deprecate the use of the Kubernetes controller manager to interact with underlying cloud platforms in favor of using cloud controller managers. As a result, there is no plan to add Kubernetes controller manager support for any new cloud platforms.
+
+This release introduces the General Availability of using a cloud controller manager for Google Cloud Platform.
+
+To learn more about the cloud controller manager, see the link:https://kubernetes.io/docs/concepts/architecture/cloud-controller/[Kubernetes Cloud Controller Manager documentation].
+
+To manage the cloud controller manager and cloud node manager deployments and lifecycles, use the Cluster Cloud Controller Manager Operator.
+
+For more information, see the xref:../operators/operator-reference.adoc#cluster-cloud-controller-manager-operator_cluster-operators-ref[Cluster Cloud Controller Manager Operator] entry in the _Platform Operators reference_.
```

Comment:
The xref points to a page titled Cluster Operators reference, should that be the text in the italics here?
```suggestion
For more information, see the xref:../operators/operator-reference.adoc#cluster-cloud-controller-manager-operator_cluster-operators-ref[Cluster Cloud Controller Manager Operator] entry in the _Cluster Operators reference_.
```

Reply:
That's what I get for copy-pasting from last version. Thanks!